### ISSUE TITLE
License warning screen comes up unexpectedly and number of "To:" rows restricted

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#67]
Since SmartTemplate v 2.11 on some systems the license messagebox comes up during Composer startup even though the user has a Standard License. It also came up for them with an updated Pro license. Apparently it also restricts the number of "to:" address rows to a single row.

**Comment by RealRaven2000** [comment]
After looking at some log files, one problem that can be seen is a broken initialisation for the new "change template" button, if it is not configured to be on the composer toolbar:

```
templatePopup is null    -    smartTemplate-composer.js:69

```

Here is a prerelease version 2.11.1 prerelease 2 to deal with this problem:

[smartTemplate-fx-2.11.1pre2.zip](https://github.com/RealRaven2000/SmartTemplate4/files/4778996/smartTemplate-fx-2.11.1pre2.zip)

Top install - please unpack the contained `smartTemplate-fx-2.11.1pre2.xpi` file to a hard drive / desktop and then drag the xpi into Thunderbird Add-ons manager.

**Comment by stephankoenig** [comment]
Just to clarify: The problem is not a single row "To:" row.
The problem is that mail.compose.addresswidget.numRowsShownDefault gets ignored.
So if it has one address row it shows one. But if you have many, it shows all. And that makes composing an e-mail or answering one impossible.

It does not happen with 2.10.1 and 2.11.1pre2!

**Comment by RealRaven2000** [comment]
I think I wasn't able to reproduce this problem because I use "copy Sent to current" which may manage the address widgets itself; give me a few days to test this. Shouldn't be too hard to find from the diff set anyway.

**Comment by RealRaven2000** [comment]
Here is the latest release - this should fix it:

[smartTemplate-fx-2.11.1.zip](https://github.com/RealRaven2000/SmartTemplate4/files/4820849/smartTemplate-fx-2.11.1.zip)

unpack the extension to hard drive before you install it.

**Comment by RealRaven2000** [comment]
Fixed in 2.11.1 - released June 21st
